src/main/check.cpp

```diff
@@ -55,7 +55,7 @@ int main(int argc, char** argv) {
     std::string asmfile;
     app.add_option("--asm", asmfile, "Print disassembly to FILE")->type_name("FILE");
     std::string dotfile;
-    app.add_option("--dot", dotfile, "Export cfg to dot FILE")->type_name("FILE");
+    app.add_option("--dot", dotfile, "Export control-flow graph to dot FILE")->type_name("FILE");
```

Comment:
I think that within this project, the term CFG should be obvious. I don't feel it helps reader to see three words everywhere for a term that is so fundamental to the project.

Reply:
For such high level visible things as command line usage, having acronyms expanded makes this project more approachable to newcomers